standard/standard/clause_7_normative_text.adoc

```diff
@@ -89,305 +84,261 @@ The Basic and Advanced Standardization Targets differ in the level of options an
+description:: Global SDU Requirements
+requirement:: /req/target_dependency
+requirement:: /req/sdu_general
+requirement:: /req/lm_general
+====
 
 The individual global requirements are defined as follows:
 
 [[global]]
-[width="100%",cols="4,6"]
-|===
-<s|Requirement:  s|G-Target
-
-<s|ID |<<req_global,/req/target_dependency>>
-2+<|There SHALL be no dependency between or among the individual Standardization Targets.
-
-|===
+[requirement]
+====
+[%metadata]
+label:: /req/target_dependency
+description:: There SHALL be no dependency between or among the individual Standardization Targets.
+====
 
 [[sdu_confomance]]
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|G-SDU 
-
-<s|ID |<<req_sdu_general,/req/sdu_general>>
-2+<|Implementations using encoded SDUs SHALL conform to the logical description of the Logical Model elements with the "Structural Data Unit - SDU" stereotype. 
-|===
-
+[requirement]
+====
+[%metadata]
+label:: /req/sdu_general
+description:: Implementations using encoded SDUs SHALL conform to the logical description of the Logical Model elements with the "Structural Data Unit - SDU" stereotype.
+====
 
 [[lm_confomance]]
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|G-LM 
-
-<s|ID |<<req_lm_general,/req/lm_general>>
-2+<|Implementations of concrete data conforming to this standard SHALL conform to all dependent or inherited classes, attributes, and associations, multiplicities, and data types in the Logical Model.
-
-|===
+[requirement]
+====
+[%metadata]
+label:: /req/lm_general
+description:: Implementations of concrete data conforming to this standard SHALL conform to all dependent or inherited classes, attributes, and associations, multiplicities, and data types in the Logical Model.
+====
 
 ==== Requirements for Standardization Target 1: Basic-YPR
 
 *Summary (Informative)*
 
 The Basic-YPR Target has a simple structure with no options. Position is specified as a point in an LTP-ENU frame and rotation is specified by yaw, pitch, and roll angles specified in decimal degrees.
 
-*Structure (Normative)*
+*Structure*
 
 [[basic_ypr_sdu]]
 .Basic YPR SDU
-image::diagram/Basic-YPR_SDU.PNG[align="center"]
-
-*Requirements Class*
-
-|===
-<|*Requirements Class:* <|*Basic-YPR* 
-
-<s|ID |<<req_basic_ypr_sdu,/req/basic_ypr/sdu>>
-<s|Target Type|Implemntation of Logical Model
-<s|Description| Basic-YPR Logical Model SDU
-<s|Dependency| /req/global
-<s|Requirement B-YPR-SDU|/req/basic_ypr/sdu
-<s|Requirement B-TP-Lon-SDU|/req/tangent_plane_longitude/sdu 
-<s|Requirement B-TP-Lat-SDU|/req/tangent_plane_latitude/sdu 
-<s|Requirement B-TP-h-SDU|req/tangent_plane_h/sdu 
-<s|Requirement B-YPR-ANGLES-SDU|/req/orientation_ypr_angles/sdu 
-
-|===
+image::diagram/Basic-YPR_SDU.PNG[]
+
+
+[[req_class_basic_ypr_sdu]]
+[requirements_class]
+====
+[%metadata]
+label:: /req/basic_ypr/sdu
+subject:: Implementation of Logical Model
+description:: Basic-YPR Logical Model SDU
+inherit:: /req/global
+requirement:: /req/basic_ypr/sdu
```

Comment:
The label of this Requirements Class (`/req/basic_ypr/sdu`) is duplicated in one of its Requirements.